packages/medusa/src/strategies/batch-jobs/product/export.ts

```diff
@@ -37,6 +36,7 @@ export default class ProductExportStrategy extends AbstractBatchJobStrategy {
   protected readonly productService_: ProductService
   protected readonly fileService_: IFileService
   protected readonly featureFlagRouter_: FlagRouter
+  protected readonly remoteQuery_: any
```

Comment:
todo: We have a type for the remoteQuery if I am correct

Reply:
will address this in a follow up PR, saw this too late.

Reply:
@riqwan have you done the followup pr?